modules/rhcos-restore-node-scan-with-stackrox-scanner.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
+// * cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-restore-node-scan-stackrox-scanner_{context}"]
+= Restoring {op-system} node scanning with the StackRox Scanner
+
+[role="_abstract"]
+If you use {ocp}, you can enable scanning of {op-system-first} nodes for vulnerabilities by using {rh-rhacs-first}.
+This feature is available with both the StackRox Scanner and Scanner V4.
+Follow this procedure if you want to use the StackRox Scanner to scan {op-system-first} nodes,
+but you want to keep using Scanner V4 to scan other nodes.
+
+.Prerequisites
+* For scanning {op-system} node hosts of the secured cluster, you must have installed Secured Cluster services on {ocp} {ocp-supported-version} or later. For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix]. For life cycle support information for {product-title-short}, see the link:https://access.redhat.com/support/policy/updates/rhacs[{product-title} Support Policy].
+
+.Procedure
+
+To enable node indexing, also known as node scanning, by using Scanner V4:
```

Comment:
should this be "by using the StackRox Scanner"?

Reply:
Yes! It should be fixed now.